bin/chflags/chflags.c

```diff
@@ -182,9 +182,11 @@ main(int argc, char *argv[])
 		if (newflags == p->fts_statp->st_flags)
 			continue;
 		if (chflagsat(AT_FDCWD, p->fts_accpath, newflags,
-		    atflag) == -1 && !fflag) {
-			warn("%s", p->fts_path);
-			rval = 1;
+		    atflag) == -1) {
+			if (!fflag) {
+				warn("%s", p->fts_path);
+				rval = 1;
```

Comment:
Help me understand... If -f is set, the fflag is set, and we'll not set rval=1 and we'll still have the bad return code this bug is supposed to fix. What am I missing?

Reply:
I believe the notion is that if both -v and -f are specified, and the chflags operation fails, the file name shouldn't be printed. Though, the documentation doesn't really say one way or another what should happen here. This change isn't obviously right IMO.

Reply:
> Help me understand... If -f is set, the fflag is set, and we'll not set rval=1 and we'll still have the bad return code this bug is supposed to fix. What am I missing?

Fixed.